src/Idris/AbsSyntaxTree.hs

```diff
@@ -474,6 +482,16 @@ parserCats = [IParse]
 elabCats :: [LogCat]
 elabCats = [IElab]
 
+loggingCatsStr :: String
+loggingCatsStr = unlines
+    [ (strLogCat IParse)
+    , (strLogCat IElab)
+    , (strLogCat ICodeGen)
+    , (strLogCat IErasure)
+    , (strLogCat ICoverage)
+    , (strLogCat IIBC)
+    ]
```

Comment:
What was unconstructive about @Melvar's comment? I can see it being considered a bit abrasive/rude (though I'm not sure I'd see it that way myself), but it's very clear on what it suggests should be done to fix the problem identified. And it seems a bit odd to decide not to make a very quick improvement to your code because the person who suggested it did so rudely.